Formula/logstash.rb

```diff
@@ -19,12 +19,13 @@ def install
       cd "tar"
     end
 
-    inreplace %w[bin/logstash], %r{^\. "\$\(cd `dirname \$SOURCEPATH`\/\.\.; pwd\)\/bin\/logstash\.lib\.sh\"}, ". #{libexec}/bin/logstash.lib.sh"
+    inreplace %w[bin/logstash], %r{^\. "\$\(cd `dirname \${SOURCEPATH}`\/\.\.; pwd\)\/bin\/logstash\.lib\.sh\"}, ". #{libexec}/bin/logstash.lib.sh"
     inreplace %w[bin/logstash-plugin], %r{^\. "\$\(cd `dirname \$0`\/\.\.; pwd\)\/bin\/logstash\.lib\.sh\"}, ". #{libexec}/bin/logstash.lib.sh"
     inreplace %w[bin/logstash.lib.sh], /^LOGSTASH_HOME=.*$/, "LOGSTASH_HOME=#{libexec}"
     libexec.install Dir["*"]
     bin.install_symlink libexec/"bin/logstash"
```

Comment:
You'll want to replace the two `bin.install_symlink libexec/"bin/logstash"` and `bin.install_symlink libexec/"bin/logstash-plugin"` with

```
bin.install libexec/"bin/logstash", libexec/"bin/logstash-plugin"
```
Then bin.env_script_all_files puts them back in libexec/bin (where they started!) and replaces them with wrapper scripts.

Reply:
Thanks, I pushed a commit for that.